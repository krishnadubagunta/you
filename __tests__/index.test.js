/* eslint-env jest */

import { shallow } from 'enzyme'
import React from 'react'
import renderer from 'react-test-renderer'

import Index from '../pages/index.js'

describe('With Enzyme', () => {
  it('Index shows "Title"', () => {
    const app = shallow(<Index />)

    expect(app.find('h1').text()).toEqual('Title')
  })
})

describe('With Snapshot Testing', () => {
  it('App shows "Title"', () => {
    const component = renderer.create(<Index />)
    const tree = component.toJSON()
    expect(tree).toMatchSnapshot()
  })
})