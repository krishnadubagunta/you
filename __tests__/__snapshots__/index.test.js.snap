// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`With Snapshot Testing App shows "Hello world!" 1`] = `
<nav
  className="sc-bdVaJa iWZHCN"
>
  <h1
    className="sc-bwzfXH gxBMIz"
  >
    Title
  </h1>
</nav>
`;

exports[`With Snapshot Testing App shows "Title" 1`] = `
<nav
  className="sc-bdVaJa iWZHCN"
>
  <h1
    className="sc-bwzfXH gxBMIz"
  >
    Title
  </h1>
</nav>
`;
